starter-workflows/code-scanning/scorecards.yml

```diff
@@ -0,0 +1,69 @@
+name: Scorecards supply-chain security
+on: 
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    # Weekly on Saturdays.
+    - cron: '30 1 * * 6'
+  push:
+    branches: [ $default-branch ]
+
+# Declare default permissions as read only.
+permissions: read-all
+
+jobs:
+  analysis:
+    name: Scorecards analysis
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed to upload the results to code-scanning dashboard.
+      security-events: write
+    
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v1
+
+      - name: "Run analysis"
+        # TODO: update with a hash once we have a release.
+        uses: ossf/scorecard-action@feat/rempol
+        with:
+          results_file: results.sarif
+          results_format: sarif
+          # For the token, 
+          # 1. Create a PAT token at https://github.com/settings/tokens/new
+          #    with the following read permissions:
+          #     - Note: SCORECARD_TOKEN (this is the name)
+          #     - Expiration: No expiration
+          #     - Scopes: 
+          #       * repo > public_repo
+          #       * admin:org > read:org
+          #       * admin:repo_hook > read:repo_hook
+          #       * read:discussion
+          #
+          # After creation, copy the token.
+          #
+          # 2. Create an encrypted secret at https://github.com/<org>/<repo>/settings/secrets/actions
+          #    with the following settings:
+          #     - Name: SCORECARD_TOKEN
+          #     - Value: the value of the token created in step 1 above.
+          repo_token: ${{ secrets.SCORECARD_TOKEN }}
+          publish_results: true
```

Comment:
@olivekl please comment on the comment :-)

Reply:
1) For consistency, should `read:discussion` say `write:discussion > read:discussion`?

2) "After creation, copy the token" --> change to "Create and copy the token." 

3) "Create an encrypted secret at..." --> change to "Create a new repository secret at..." (to match the wording on that page)

Reply:
Thank you!